### PR TITLE
Several fixes including a fix for desync after instabilities, camera glitch cause by queue, a fix for ignition aswell as hazard lights

### PR DESCRIPTION
**Pull request by Olrosse**
also fix for remote vehicle turning local, fix for unicycle instability loop and more, the individual commits should have most of the change-log

and added an experimental option to disable instability pausing, it is toggleable live in game and automatically reverts when leaving a server

i might PR a rewrite of the ignition code that is more robust later, it is just a bit messy right now so i will try to simplify it a bit first

### CODE REVIEW

File: lua/ge/extensions/MPVehicleGE.lua [hunk 1]
```diff
@@ -887,17 +887,17 @@ local function applyQueuedEvents()
...
 		UI.updateQueue(vehicleSpawnQueue or {}, vehicleEditQueue or {})
 	end
 
 	--if not vehicleEditQueue then return end
 	--dump(vehicleEditQueue)
 	for vehicleID, edit in pairs(vehicleEditQueue) do
 		log('I', "applyQueuedEvents", "Applying queued edit")
-		applyVehEdit(vehicleID, edit)
 		vehicleEditQueue[vehicleID] = nil
+		applyVehEdit(vehicleID, edit)
```
Comment (Starystars67) [inline lua/ge/extensions/MPVehicleGE.lua:900]: Why did you do the apply after removal from the array? Not complaining just trying to understand your theory.

File: lua/vehicle/extensions/BeamMP/MPElectricsVE.lua [hunk 2]
```diff
@@ -462,8 +462,8 @@ local function applyElectrics(data)
 		end
 
 		-- Ignition syncing
-		if decodedData.ignition ~= nil then
-			if electrics.values.ignition ~= decodedData.ignition then
+		if decodedData.ignition ~= nil and electrics.values.ignition ~= nil then
```
Comment (Starystars67) [inline lua/vehicle/extensions/BeamMP/MPElectricsVE.lua:465]: Good find, Why not use `Number()` or similar? to convert it to a number rather than Boolean?

Comment (Starystars67) [reply lua/vehicle/extensions/BeamMP/MPElectricsVE.lua:465]: Might allow for finer control?

**Review by Starystars67 — COMMENTED** [review]
Nice work, Just a couple of comments if you could please address them?

File: lua/vehicle/extensions/BeamMP/MPElectricsVE.lua (hunk 2 above)
Comment (Olrosse) [reply lua/vehicle/extensions/BeamMP/MPElectricsVE.lua:465]: Do you happen to know why the data is received as a boolian in the first place when the electric number locally is a number?

Comment (Olrosse) [reply lua/vehicle/extensions/BeamMP/MPElectricsVE.lua:465]: Fair enough :) yeah i think it is good for mering, unless you have anything else to note?

**Review by Starystars67 — APPROVED** [review]

**Review by lionkor — APPROVED** [review]
Looks great, can't wait to try this!

**Review by 20dka — APPROVED** [review]

**Pull request merged**